diagnostics/ENSO_RWS/LEVEL_01/get_flux_clima.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np

```

Comment:
get_flux_clima.py, get_flux_in.py, and get_nino_index.py are very similar, and can probably be combined into one script by modifying the arguments.